client/question/question.js

```diff
@@ -334,6 +334,20 @@ tie.constant('WRONG_LANGUAGE_ERRORS', {
     ],
     allowMultiline: false
   }, {
+    // Used the wrong casing for "True" or "False".
+    errorName: 'booleans',
+    regExString: '\\b(true|TRUE|false|FALSE)\\b',
+    feedbackParagraphs: [
+      {
+        type: 'text',
+        content: [
+          "Note that, in Python, we write boolean values as True and False. ",
+          "Triple-check to make sure you've got the right casing!"
```

Comment:
"Double-check to make sure 'True' or 'False' are written with the letters in the same case."

Reply:
Basically, please add a little context around "casing" or "case" to make sure people don't get thrown off.

Reply:
Good catch, thanks! Done.